posts/2024-10-02-gccrs-an-alternative-compiler-for-rust.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: gccrs: An alternative compiler for Rust
```

Comment:
To make this correct YAML:

```suggestion
title: "gccrs: An alternative compiler for Rust"
```

Reply:
thanks! fixed in [cf1fa41](https://github.com/rust-lang/blog.rust-lang.org/pull/1412/commits/cf1fa414f5ee0abd8b0c70713715c15e09c74f27) and rebased on latest changes